ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/dropdown.html

```diff
@@ -0,0 +1,52 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<sly data-sly-use.renderer="${'dropdown.js'}"
+     data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-use.label="${renderer.labelPath}"
+     data-sly-use.shortDescription="${renderer.shortDescriptionPath}"
+     data-sly-use.longDescription="${renderer.longDescriptionPath}"
+     data-sly-use.errorMessage="${renderer.errorMessagePath}"
+     data-sly-use.questionMark="${renderer.questionMarkPath}"></sly>
+<div data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+     data-sly-use.dropdown="com.adobe.cq.forms.core.components.models.form.DropDown"
+     data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
+     class="cmp-adaptiveform-dropdown"
+     data-cmp-is="adaptiveFormDropDown"
+     id="${component.id}"
+     data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
+    <div data-sly-call="${label.label @componentId=component.id, labelValue=dropdown.label.value, labelVisible=dropdown.label.visible, bemBlock='cmp-adaptiveform-dropdown'}" data-sly-unwrap></div>
+    <div data-sly-call="${questionMark.questionMark @componentId=component.id, longDescription=dropdown.description, bemBlock='cmp-adaptiveform-dropdown'}" data-sly-unwrap></div>
+    <select class="cmp-adaptiveform-dropdown__widget"
+            aria-label="${dropdown.label.visible && dropdown.label.value}"
+            name="${dropdown.name}"
+            readonly="${dropdown.readOnly}"
+            required="${dropdown.required}"
+            data-sly-attribute.multiple="${dropdown.isMultiSelect ? 'multiple' : ''}"
+            data-cmp-required-message="${dropdown.constraintMessages.required}">
```

Comment:
This attribute is not required, since runtime model of form already has these details. This is redundant

Reply:
This was added so that in authoring view the component shows a list when user chooses multi-select.